code/modules/admin/admin.dm

```diff
@@ -1086,6 +1086,9 @@ var/global/noir = 0
 						H.set_mutantrace(/datum/mutantrace/flubber)
 					if ("Cow")
 						H.set_mutantrace(/datum/mutantrace/cow)
+					if ("Dogman")
+						H.set_mutantrace(/datum/mutantrace/doggo)
+						. = 1
```

Comment:
Isn't used by the two above, is this just debug code? (I see it messages the admins below, which seems unneeded.)

Reply:
Personally, since the rest of the list seems to have that line, including the lizard entry, I thought it should've been there for the other two that lack it. I didn't add it to those two because I didn't think it'd be within the scope of this PR. I can remove it if it shouldn't be there.

Reply:
Good shout, I didn't look back further than the window shown by GitHub. I've no issue with you adding `. = 1` to the two above as part of this PR. A little code cleanliness in a PR isn't awful as long as it's related, and this is.